bridge-token-factory/src/lib.rs

```diff
@@ -28,24 +27,24 @@ const BRIDGE_TOKEN_BINARY: &'static [u8] = include_bytes!(std::env!(
 const BRIDGE_TOKEN_INIT_BALANCE: Balance = 3_000_000_000_000_000_000_000_000; // 3e24yN, 3N
 
 /// Gas to initialize BridgeToken contract.
-const BRIDGE_TOKEN_NEW: Gas = 10_000_000_000_000;
+const BRIDGE_TOKEN_NEW: Gas = Gas(10_000_000_000_000);
 
 /// Gas to call mint method on bridge token.
-const MINT_GAS: Gas = 10_000_000_000_000;
+const MINT_GAS: Gas = Gas(10_000_000_000_000);
 
 /// Gas to call finish deposit method.
 /// This doesn't cover the gas required for calling mint method.
-const FINISH_DEPOSIT_GAS: Gas = 30_000_000_000_000;
+const FINISH_DEPOSIT_GAS: Gas = Gas(30_000_000_000_000);
 
 /// Gas to call finish update_metadata method.
-const FINISH_UPDATE_METADATA_GAS: Gas = 5_000_000_000_000;
+const FINISH_UPDATE_METADATA_GAS: Gas = Gas(5_000_000_000_000);
 
 /// Amount of gas used by set_metadata in the factory, without taking into account
 /// the gas consumed by the promise.
-const OUTER_SET_METADATA_GAS: Gas = 15_000_000_000_000;
+const OUTER_SET_METADATA_GAS: Gas = Gas(15_000_000_000_000);
 
 /// Amount of gas used by bridge token to set the metadata.
-const SET_METADATA_GAS: Gas = 5_000_000_000_000;
+const SET_METADATA_GAS: Gas = Gas(5_000_000_000_000);
```

Comment:
Please use `near-sdk`'s constants for `Gas::TGas` and `ONE_NEAR` (see my comment below)

Reply:
Fixed 8e2b316371f19907869b1bd8506f976beb82ce31